docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -130,11 +134,6 @@ Preparing For a Release
       mkvirtualenv flocker-release-${VERSION}
       pip install --editable .[release]
 
-#. Back port features from master (optional)
```

Comment:
I think that the maintenance branch makes this unnecessary. If so, I'll close the issue to document the process.